integrations/terraform/testlib/terraform_oss_test.go

```diff
@@ -0,0 +1,65 @@
+package testlib
+
+import (
+	eauth "github.com/gravitational/teleport/e/lib/auth"
+	"github.com/gravitational/teleport/integrations/lib/testing/integration"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/modules"
+	"github.com/gravitational/teleport/lib/plugin"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+func TestTerraformOSS(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{
+		TestFeatures: modules.Features{
+			OIDC:                    true,
+			SAML:                    true,
+			AdvancedAccessWorkflows: true,
+		},
+	})
+
+	authPlugin, err := eauth.NewPlugin(eauth.Config{
+		License: eauth.ValidLicense{},
+	})
```

Comment:
I don't think we should depend on `e` code when running the OSS test suite

Reply:
fixed, I mistakenly exchanged between the OSS and Enterprise suite tests 🙃